### ISSUE TITLE
User interface (UI) not working with HDR on

### ISSUE THREAD

**Issue by herrsocke** [yuliskov/SmartTube#1986]
Hi,
I experience the problem that the interface does not work when HDR is on. It gets extremly bright so all you see is white.

[https://i.imgur.com/3L4evjA.jpg](https://i.imgur.com/3L4evjA.jpg)

This problem does not occur when I am using the normal youtube app. the device I  use is LG OLED55CX9LA.

**Comment by jkogen** [comment]
Same here on Philips 65OLED706/12. Its only on HDR vids like from the linked channel: https://youtu.be/AR3eD_O0q80 
Impossible to control these vids

**Comment by astraetech** [comment]
Yep, same here on a firestick 4k max and a HiSense u7h. Latest STN Beta. The bottom part of the video player UI is all whitened out. Happens only on some videos, may be related to Dolby Vision IQ?

**Comment by jkogen** [comment]
I am also using FireTV-stick 4k. As far as I know SmartTube on my TV OS doesn't have this Problem.

----------------------------------------

11.12.2022 08:30:34 astraetech ***@***.***>:
